cypress/integration/learn/challenges/multifile.js

```diff
@@ -0,0 +1,15 @@
+const location =
+  '/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-2';
+
+describe('Challenge with multifile editor', () => {
+  before(() => {
+    cy.visit(location);
+  });
+
+  it('renders the file tab buttons', () => {
+    cy.wait(5000);
```

Comment:
I don't think cy.wait is a good idea here.  Reason being, we're already waiting for something specific to be in the page (the tabs). If they're taking a while to appear, then we can extend the timeout for the tardy element.

Also, the error is in the before hook, because this location isn't getting built. And *that* is because I keep forgetting to flip `SHOW_NEW_CURRICULUM` to `true` in sample.env.  Ahem.
```suggestion
```

Reply:
I've been hoodwinked

Reply:
So do we want to update the sample.env file? Or should we set the flag in the workflow? How do we want to approach this?

Reply:
I think it's reasonable to update sample.env now.  It probably should have happened when we released the beta version, but there was a lot going on.

That said, I'm not 100% sure what will happen with all the tests, particularly the other languages. We might as well flip it to true and see what happens.

Reply:
Here goes nothing!